llvm/utils/lit/tests/Inputs/shtest-cat/cat-v.txt

```diff
@@ -0,0 +1,49 @@
+# C Program to generate all ASCII characters in allchars.txt
```

Comment:
```suggestion
## C Program to generate all ASCII characters in allchars
```
The file is `allchars`. If you change the extension, please match it.

Reply:
done